src/config/dependencies.py

```
@@ -45,12 +47,12 @@ def get_jwt_auth_manager(settings: BaseAppSettings = Depends(get_settings)) -> J
     return JWTAuthManager(
         secret_key_access=settings.SECRET_KEY_ACCESS,
         secret_key_refresh=settings.SECRET_KEY_REFRESH,
-        algorithm=settings.JWT_SIGNING_ALGORITHM
+        algorithm=settings.JWT_SIGNING_ALGORITHM,
```

The JWTAuthManager is created directly from settings here. If there's any chance `settings.JWT_SIGNING_ALGORITHM` (or the secret keys) could be missing or mis-typed, consider validating or failing fast with a clear error message. That will produce a clearer startup-time error instead of an obscure exception at first token operation.